backbone/src/main/java/org/researchstack/backbone/model/survey/SurveyItem.java

```diff
@@ -35,18 +35,6 @@
     @SerializedName("items")
     public List<T> items;
 
-    @SerializedName("skipIdentifier")
-    public String skipIdentifier;
-
-    @SerializedName("skipIfPassed")
-    public boolean skipIfPassed;
-
-    // TODO: implement this?
-    public String rulePredicate;  // this is an NSPredicate on iOS, how do we convert?
```

Comment:
Predicate class requires API level 24 or higher.  But also, it really isn't necessary, iOS was using it in an odd way.  The way I did NavigableSteps I think is more understandable anyways.